nms/app/context/CbsdContext.tsx

```diff
@@ -174,22 +180,20 @@ export function CbsdContextProvider({
     cbsds: [],
     total_count: 0,
   });
-  const [paginationOptions, setPaginationOptions] = useState<{
-    page: number;
-    pageSize: number;
-  }>({
-    page: 0,
+  const [paginationOptions, setPaginationOptions] = useState<PaginationOptions>({
+    page: -1,
```

Comment:
Is `-1` instead of `0` valid input?

Reply:
No :) it was a hack. I've figured out how to make it properly now :)
So now initial page is `0`.